client/src/pages/TotalExpensesChart.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { Typography, Divider } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import Paper from "@material-ui/core/Paper";
+import {
+  Chart,
+  ArgumentAxis,
+  ValueAxis,
+  SplineSeries
+} from "@devexpress/dx-react-chart-material-ui";
+
+const useStyles = theme => ({
+  root: {
+      flexGrow: 1
+  }
+});
+
+class TotalExpensesChart extends Component {
+  state = {
+      daily_expenses: [{expense: 10.01, date: "Nov 3"},
+                       {expense: 3.59, date: "Nov 4"},
+                       {expense: 5.13, date: "Nov 5"}]
+  }
+
+  componentDidMount() {
+      fetch("/receipts/daily-expenses/2019/11")
```

Comment:
maybe by default you can find out the current year and current month?

Reply:
you're right! I'll have a default value for the chart data added in!